climada/engine/test/test_impact_calc.py

```diff
@@ -101,6 +101,44 @@ def test_apply_cover_to_mat(self):
             imp.todense(), np.array([[0, 0, 1], [0, 1, 0]])
             )
 
+    def test_error_handling_mismatch_haz_type(self):
+        """Test error handling in case hazard type of hazard
+        does not appear in impf_set or exposures"""
+        exp_haz_correct = deepcopy(ENT.exposures)
+        exp_haz_wrong = deepcopy(ENT.exposures)
+        exp_haz_wrong.gdf.rename(columns={'impf_TC': 'impf_WS'}, inplace=True)
```

Comment:
It would be better to create an empty exposures here I think. 
```suggestion
        exp_haz_correct = Exposures()
        exp_haz_wrong.gdf.rename(columns={'impf_': 'impf_TC'}, inplace=True)
        exp_haz_wrong = Exposures()
        exp_haz_wrong.gdf.rename(columns={'impf_': 'impf_WS'}, inplace=True)
```

Reply:
Also, I would maybe rename `exp_haz_correct` to `exp_haz_tc` or so?

Reply:
I now create an empty hazard and empty exposures. Also I renamed the test objects to `haz_tc`, `exp_tc`, `exp_ws`, `impfset_tc`, `impfset_ws`, `impfset_undef` to clarifiy what is being tested.